src/common/transformations/src/transformations/control_flow/unroll_if.cpp

```diff
@@ -54,14 +54,7 @@ bool ov::pass::UnrollIf::run_on_model(const std::shared_ptr<ov::Model>& f) {
         }
         for (const auto& output_desc : output_descriptions) {
             std::shared_ptr<ov::op::v0::Result> result = body->get_results()[output_desc->m_body_value_index];
-            const auto& in_value = result->input_value(0);
 
-            // set output name to Tensor to store it for openvino to cnn conversion
-            OPENVINO_SUPPRESS_DEPRECATED_START
-            ov::descriptor::set_ov_tensor_legacy_name(
```

Comment:
should `set_ov_tensor_legacy_name` (and probably, `get_ov_tensor_legacy_name` as well, if it won't be used as all `set_ov_tensor_legacy_name` usages are removed) be removed from OpenVINO code base as well?

Reply:
Looks like `legacy name` can be removed independently
#27307